.bazelci/presubmit.yml

```diff
@@ -4,22 +4,48 @@ tasks:
   ubuntu1604:
     name: ":linux:"
     shell_commands:
-    - yes | ./configure
+    - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    - bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
+    - source $HOME/miniconda/bin/activate
```

Comment:
No need for this source

Reply:
Fixed in commit [`814cd24`](https://github.com/plaidml/plaidml/pull/435/commits/814cd246c8e9fee003c7abba4b074f7e7dc9259d)